tests/bluetooth/bsim/custom_ltk/testcase.yaml

```diff
@@ -0,0 +1,10 @@
+tests:
+  bluetooth.custom_ltk:
+    build_only: true
+    tags:
+      - bluetooth
+    platform_allow:
+      - nrf52_bsim/native
+    harness: bsim
```

Comment:
does that work; or should we use to ass a functional harness to be able run the test scripts?